docs/contribution/30-overview/1-extensions.mdx

```diff
@@ -89,3 +115,53 @@ You are responsible for versioning your extensions and may introduce individual
 
 If the extension provides an API used by external applications, you should ensure its stability.
 To this end, you may use commonplace conventions and best practices for API versioning.
+
+## Adding an extension to a Context
+
+1. A **Contributor** publishes an extension.
+2. An end user adds the **Extension** to a **Context**, creating an **ExtensionInstance**.
+3. If the **Extension** is free, it becomes usable immediately.
+4. If it is not free, a checkout via Stripe must be completed.
+  - There is a grace period after which the **ExtensionInstance** is deactivated until the user completes the purchase and the **ExtensionInstance** reactivates itself.
+    After the purchase, the user can use the **Extension**.
+5. If an already published **Extension** is assigned a price, the grace period for the customer to complete the checkout starts immediately.
+
+If an already published extension is assigned a price, the grace period for the customer to complete the checkout starts immediately.
```

Comment:
You should consider to rewrite bullet points 4 and 5. This sentence seems to be a duplicate and overall i find it hard to understand what you are trying to explain